modules/backend/formwidgets/taglist/partials/_taglist.php

```diff
@@ -19,20 +19,11 @@
             <li class="taglist__item"><?= e(trans($option)) ?></li>
         <?php endforeach ?>
     </ul>
-    <?php if (is_array($field->value)): ?>
-        <?php foreach ($displayOnlyOptions as $option): ?>
-        <input
-            type="hidden"
-            name="<?= $field->getName() ?>[]"
-            value="<?= $option ?>">
-        <?php endforeach ?>
-    <?php else: ?>
-        <input
-            type="hidden"
-            name="<?= $field->getName() ?>"
-            value="<?= $field->value ?>">
-    <?php endif ?>
```

Comment:
Why was this removed? Isn't it needed for trigger support in preview / readonly contexts?

Reply:
Ah so that's what it's for. I'll re-add.